config.py

```diff
@@ -0,0 +1,122 @@
+# flake8: noqa
+import os
+
+
+class Config(object):
+
+    NAME = os.getenv('RAS-PARTY', 'ras-party')
+    VERSION = os.getenv('VERSION', '1.0.0')
+    SCHEME = os.getenv('http')
+    HOST = os.getenv('HOST', '0.0.0.0')
+    PORT = os.getenv('PORT', 8081)
+    DEBUG = os.getenv('DEBUG', False)
+    LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
+    SECRET_KEY = os.getenv('SECRET_KEY', 'aardvark')
+    EMAIL_TOKEN_SALT = os.getenv('EMAIL_TOKEN_SALT', 'aardvark')
+    DATABASE_SCHEMA = os.getenv('DATABASE_SCHEMA', 'ras-party')
+    DATABASE_URI = os.getenv('DATABASE_URI', "sqlite:///:memory:")
+    REQUESTS_GET_TIMEOUT = os.getenv('REQUESTS_GET_TIMEOUT', 20)
+    REQUESTS_POST_TIMEOUT = os.getenv('REQUESTS_POST_TIMEOUT', 20)
+    SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME', 'dummy_user')
+    SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD', 'dummy_password')
+    SECURITY_REALM = os.getenv('SECURITY_REALM', 'sdc')
+
+    # dependencies
+
+    RAS_PARTY_DATABASE_SCHEMA = os.getenv('RAS_PARTY_DATABASE_SCHEMA', 'partysvc')
+    RAS_PARTY_DATABASE_URI = os.getenv('RAS_PARTY_DATABASE_URI', 'postgres://postgres:postgres@localhost:5432/postgres')
+
+    RAS_PUBLIC_WEBSITE_PROTOCOL = os.getenv('RAS_OAUTH_SERVICE_PROTOCOL', 'http')
```

Comment:
`RAS_PUBLIC_WEBSITE_PROTOCOL` should be set to it's equivalent environment variable, Not `RAS_OAUTH_SERVICE`.
Same with the two lines below

Reply:
Knew that copy-pasta would bite me eventually...